frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -300,15 +311,29 @@ end
 ReaderCoptListener.onCloseDocument = ReaderCoptListener.unscheduleHeaderRefresh
 ReaderCoptListener.onSuspend = ReaderCoptListener.unscheduleHeaderRefresh
 
+function ReaderCoptListener:addAdditionalHeaderContent(content_func)
+    table.insert(self.external_header_content, content_func)
```

Comment:
Better settle on one word : `external` or `additional` - and name the methods as the array where we store stuff.